tests/prismicio.AspNetCore.Tests/DocTest.cs

```diff
@@ -6,139 +6,151 @@
-			var document = Document.Parse(JObject.Parse(json));
-			// startgist:dabf36e591c93029440a:prismic-embed.cs
-			var video = document.GetEmbed ("article.video");
-			// Html is the code to include to embed the object, and depends on the embedded service
-			var html = video.Html;
-			// endgist
-			Assert.Equal("<iframe width=\"480\" height=\"270\" src=\"http://www.youtube.com/embed/baGfM6dBzs8?feature=oembed\" frameborder=\"0\" allowfullscreen></iframe>", html);
-		}
-
-		[Fact]
-		public void ColorTest()
-		{
-			var json = "{\"id\":\"abcd\",\"type\":\"article\",\"href\":\"\",\"slugs\":[],\"tags\":[],\"data\":{\"article\":{\"background\":{\"type\":\"Color\",\"value\":\"#000000\"}}}}";
-			var document = Document.Parse(JObject.Parse(json));
-			// startgist:87cc65a8d1d02f4b4342:prismic-color.cs
-			var bgcolor = document.GetColor("article.background");
-			var hex = bgcolor.Hex;
-			// endgist
-			Assert.Equal("#000000", hex);
-		}
-
-		[Fact]
-		public void GeopointTest()
-		{
-			var json = "{\"id\":\"abcd\",\"type\":\"article\",\"href\":\"\",\"slugs\":[],\"tags\":[],\"data\":{\"article\":{\"location\":{\"type\":\"GeoPoint\",\"value\":{\"latitude\":48.877108,\"longitude\":2.333879}}}}}";
-			var document = Document.Parse(JObject.Parse(json));
-			fragments.GeoPoint place = document.GetGeoPoint("article.location");
-			var coordinates = place.Latitude + "," + place.Longitude;
+        public void GeopointTest()
+        {
+            var json = "{\"id\":\"abcd\",\"type\":\"article\",\"href\":\"\",\"slugs\":[],\"tags\":[],\"data\":{\"article\":{\"location\":{\"type\":\"GeoPoint\",\"value\":{\"latitude\":48.877108,\"longitude\":2.333879}}}}}";
+            var document = Document.Parse(JObject.Parse(json));
+            fragments.GeoPoint place = document.GetGeoPoint("article.location");
+            var coordinates = place.Latitude + "," + place.Longitude;
             // endgist
-			#pragma warning restore CS0219 // Variable is assigned but its value is never used
+#pragma warning restore CS0219 // Variable is assigned but its value is never used
             Assert.Equal(48.877108, place.Latitude);
             Assert.Equal(2.333879, place.Longitude);
-		}
-	}
+        }
+
+        [Fact]
+        public void PublishDateTest()
+        {
+            var document = Fixtures.GetDocument("language.json");

```

Comment:
language.json ==> seem to not exist. Are you sure this is what you wanted to write ?

Reply:
I'll complete #1 and merge into this branch which will resolve this, issue